source/index.html

```diff
@@ -85,80 +85,98 @@ <h2 class="banner__title">Парковочные решения</h2>
             </div>
             <div class="user-field__user-picture">
-              <img src="#" alt="" class="user-field__img" widht="40" height="40">
+              <img src="#" alt="" class="user-field__img" height="40">
             </div>
-          </div>
-          <img src="#" alt="" class="questions__image" width="328" height="436">
-        </div>
+          </li>
+          <li class="questions__description">
+            <img src="#" alt="" class="questions__image" width="328" height="436">
+            <span class="urer-field__help">?</span>
+            <p class="questions__text">
+              Большинство (78,5%) представителей жилых комплексов не знают, какое парковочное решение оптимально
+              подойдет для их ЖК. И это нормально, так как на рынке большой выбор парковочных решений, которые могут
+              предлагать недобросовестные подрядчики
+            </p>
+          </li>
+        </ul>
       </section>
 
       <section class="benefits">
-        <span class="breadcrumb">Вот почему</span>
-        <p class="breadcrumb__text">Выбирают нас</p>
+        <h2 class="benefits__title visually-hidden">Наши Приемущества</h2>
+        <span class="tip">Вот почему</span>
+        <h2 class="tip__title">Выбирают нас</h2>
 
         <ul class="benefits__list">
           <li class="benefits__item">
             <img src="#" alt="" class="benefits__icon" width="40" height="40">
-            <h3 class="benefits__title">Индивидуально</h3>
+            <h3 class="benefits__item-title">Индивидуально</h3>
             <p class="benefits__subtitle">Широкий выбор решений под ваши необходимости</p>
           </li>
           <li class="benefits__item">
             <img src="#" alt="" class="benefits__icon" width="40" height="40">
-            <h3 class="benefits__title">Удобно</h3>
+            <h3 class="benefits__item-title">Удобно</h3>
             <p class="benefits__subtitle">Реализация проекта под ключ</p>
           </li>
           <li class="benefits__item">
             <img src="#" alt="" class="benefits__icon" width="40" height="40">
-            <h3 class="benefits__title">Профессионально</h3>
+            <h3 class="benefits__item-title">Профессионально</h3>
             <p class="benefits__subtitle">Выбор, установка, настройка и запуск парковки в вашем ЖК нашими опытными
               инженерами</p>
           </li>
           <li class="benefits__item">
             <img src="#" alt="" class="benefits__icon" width="40" height="40">
-            <h3 class="benefits__title">Надежно</h3>
+            <h3 class="benefits__item-title">Надежно</h3>
             <p class="benefits__subtitle">Гарантия на парковочную систему - 12 месяцев</p>
           </li>
         </ul>
       </section>
 
       <section class="solutions">
-        <span class="breadcrumb">Надежные</span>
-        <p class="breadcrumb__text">Готовые решения</p>
+        <h2 class="solutions__title visually-hidden">Доступные решения</h2>
```

Comment:
Its not necessary to add several classes for element which will be hidden, don't it?